### ISSUE TITLE
Trained on 256px but inference on 512px?

### ISSUE THREAD

**Issue by andreemic** [timothybrooks/instruct-pix2pix#97]
As far as I understand the training config, IP2P was trained on 256x256 images but images in the paper are generated at 512px.

1. Doesn't training at a different resolution than inference impact performance?
2. If so, why did you train on 256px? Did you try training on 512px?

**Comment by timothybrooks** [comment]
Yes! I trained at lower resolution because I did not have compute available to me to train at 512x512. I tried at 128x128 as well, and things definitely improved at 256x256, so it might improve more if trained at 512x512.

**Comment by kosmels** [comment]
Hello guys! @timothybrooks thank you for your answer. I have 1 question - do I need to change anything directly in source code (or train config) if I want to train on larger resolution, e.g. 512x512?

**Comment by stihuangyuan** [comment]
modify train image resolution to 512 happen this error:
xc = torch.cat([x] + c_concat, dim=1)
[rank0]: RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 32 but got size 64 for tensor number 1 in the list.

**Comment by khanhpham3799** [comment]
im also try to train on 512 resolution, does anyone have any resolution to change in the config code?
